msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -2,24 +2,24 @@
 title: Teams JavaScript client library
 author: heath-hamilton
 ms.author: surbhigupta
-description: In this module, learn Microsoft Teams JavaScript client library (TeamsJS), which can help you build app experiences hosted in an <iframe> in Teams, Microsoft 365 (formerly Office), and Outlook.
+description: In this module, learn TeamsJS, that can help you build app experiences hosted in an <iframe> in Teams, Microsoft 365 (formerly Office), and Outlook.
```

Comment:
In this particular case, "which" works but "that" doesn't sound quite right:

In this module, learn TeamsJS, which can help you . . .